src/components/general/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import * as styles from "./styles.less";
+
+type TabsProps = {
+    onTabClick: (tabKey: string) => void,
+    activeTabKey: string,
+    children: any,
+};
+
+const Tabs = ({ onTabClick, activeTabKey, children }: TabsProps) => {
+    const renderTabContent = () => {
+        const active = children.find(child => child.props.tabKey === activeTabKey);
```

Comment:
If the children prop only contains one child it will not be an array (and therefore .find will throw an error). Use React.Children.toArray(children) to ensure an array.